js/skin.js

```diff
@@ -267,6 +278,7 @@ var Skin = React.createClass({
               handleVrPlayerMouseUp={this.handleVrPlayerMouseUp}
               handleVrPlayerClick={this.handleVrPlayerClick}
               handleVrPlayerFocus={this.handleVrPlayerFocus}
+              areClassNamesEquals={this.areClassNamesEquals}
```

Comment:
Why do we need to pass this function to components? It would be cleaner to have components import these from Utils.

Reply:
\+